src/api/java/IDecRefQueue.java

```diff
@@ -21,7 +21,7 @@
 
 public abstract class IDecRefQueue
 {
-    protected Object m_lock = new Object();
+    protected final Object m_lock = new Object();
```

Comment:
Like Context::creation_lock: Will this break at runtime in parallel programs, or when a parallel garbage collector is used?